src/ci/docker/README.md

```diff
@@ -0,0 +1,22 @@
+# Building
```

Comment:
It would be nice with a first paragraph here that explains the *WHY* and purpose of this docker container.

This will also become more obvious once `TESTING.md` is filled out, ie this container is used to simplify the running of integration tests locally. (Manually setting up and running `ccm` the same way as travis/circleci setups it up is a PITA.)
But to repeat it here, as the first paragraph, would help set the context better for the reader!

Reply:
@michaelsembwever done